src/test/librados/aio.cc

```diff
@@ -3699,6 +3699,73 @@ TEST(LibRadosAioEC, StatRemovePP) {
   delete my_completion4;
 }
 
+TEST(LibRadosAioEC, ExecuteClass) {
+  AioTestDataEC test_data;
+  rados_completion_t my_completion;
+  ASSERT_EQ("", test_data.init());
+  ASSERT_EQ(0, rados_aio_create_completion((void*)&test_data,
+	      set_completion_completeEC, set_completion_safeEC, &my_completion));
+  char buf[128];
+  memset(buf, 0xcc, sizeof(buf));
+  ASSERT_EQ(0, rados_aio_write(test_data.m_ioctx, "foo",
+			       my_completion, buf, sizeof(buf), 0));
+  {
+    TestAlarm alarm;
+    sem_wait(test_data.m_sem);
+    sem_wait(test_data.m_sem);
+  }
+  ASSERT_EQ(0, rados_aio_get_return_value(my_completion));
+  rados_completion_t my_completion2;
+  ASSERT_EQ(0, rados_aio_create_completion((void*)&test_data,
+	      set_completion_completeEC, set_completion_safeEC, &my_completion2));
+  char out[128]
```

Comment:
add a ";" please.

@iain-buclaw-sociomantic could you try to compile the test and run it locally? you can 
```
make ceph_test_rados_api_aio
ctest -R ceph_test_rados_api_aio // with a running vstart cluster
```